### ISSUE TITLE
Customer Home: Create a checklist to help users grow their site

### ISSUE THREAD

**Issue by danhauk** [Automattic/wp-calypso#39187]
WIP

Provide guidance to users through the use of a checklist that helps them grow their site.

### Grow your site

1. Connect Google Analytics
2. Optimize your site's build-in SEO tools
3. Introduce yourself with a blog post
4. Schedule a Quick Start session

**Comment by Aurorum** [comment]
@gwwar Is this something I'm able to do? I'm well into the process of making the tasks, but I can't find a way to actually include them in a checklist; it appears lines like these have disappeared, but I'm not sure when or where.

https://github.com/Automattic/wp-calypso/blob/44284009caed1b360d9652ee2e5c457b442848b2/client/my-sites/checklist/wpcom-checklist/wpcom-task-list.js#L19-L24

**Comment by danhauk** [comment]
@Aurorum Thanks for jumping on this. This was mostly a placeholder issue while we worked on what guidance would look like post-launch. After some thinking, discussing, and designer, we've decided to move away from a series of checklists to guide people through their site. More to come later once decisions are made.

I apologize for not communicating that here. I'll go through and tidy up any issues that are no longer relevant so you don't end up doing extra work for nothing.
